engine/common/rpc/convert/convert.go

```diff
@@ -393,3 +394,70 @@ func BytesToInmemSnapshot(bytes []byte) (*inmem.Snapshot, error) {
 
 	return inmem.SnapshotFromEncodable(encodable), nil
 }
+
+func ExecResultProtoToFlowExecResult(proto *entities.ExecutionResult) (*flow.ExecutionResult, error) {
```

Comment:
a bit nit but maybe to follow the naming convention for converters this could be named `protoToExecutionResult`